README.rst

```diff
@@ -71,3 +71,23 @@ You can filter tests based on the attributes. There is a attribute named ``test_
 For running ``webidentity_test`` you'll need have Keycloak running.
 
 In order to run any STS test you'll need to add "iam" section to the config file. For further reference on how your config file should look check ``s3tests.conf.SAMPLE``.
+
+
+========================
+ Docker environment
```

Comment:
I suppose it'd be better to place the text in `NEOFS_README.md` to avoid conflicts with original `ceph` repository.

Reply:
Agree, forgot about this file.